zone_example_test.go

```diff
@@ -31,7 +31,7 @@ func ExampleAPI_ListZones_filter() {
 	}
 
 	// Fetch a slice of zones example.org and example.net.
-	zones, err := api.ListZones(1, "example.org", "example.net")
```

Comment:
This commit should be removed:

* you fix the call in a later commit
* your change no longer matches the comment